sycl/test-e2e/Matrix/joint_matrix_apply_two_matrices.cpp

```diff
@@ -0,0 +1,14 @@
+//==------ joint_matrix_apply_two_matrices.cpp  - DPC++ joint_matrix--------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: aspect-ext_intel_matrix
+
+// RUN: %{build} -o %t.out
```

Comment:
to not depend on compiler defaults regarding how nan is treated in floating point operations:

```suggestion
// RUN: %{build} -ffp-model=precise  -o %t.out
```